src/lib/libatomic.js

```diff
@@ -50,6 +50,7 @@ addToLibrary({
   /**
    * @param {number=} maxWaitMilliseconds
    */
+  /** @suppress {duplicate, checkTypes} */
```

Comment:
Merge this with the existing docstring above.

Reply:
@sbc100 done, merged with the existing docstring above.